src/main/telemetry/ghst.c

```diff
@@ -112,10 +113,42 @@ void ghstFramePackTelemetry(sbuf_t *dst)
     sbufWriteU8(dst, 0x00);                     // tbd3
 }
 
+// GPS data, primary, positional data
+void ghstFrameGpsPrimaryTelemetry(sbuf_t *dst)
+{
+    // use sbufWrite since CRC does not include frame length
+    sbufWriteU8(dst, GHST_FRAME_GPS_PAYLOAD_SIZE + GHST_FRAME_LENGTH_CRC + GHST_FRAME_LENGTH_TYPE);
+    sbufWriteU8(dst, GHST_DL_GPS_PRIMARY);
+
+    sbufWriteU32(dst, gpsSol.llh.lat);
+    sbufWriteU32(dst, gpsSol.llh.lon);
+
+    // constrain alt. from -1 km to +10 km, send in units of meters
+    const uint16_t altitude = (constrain(getEstimatedAltitudeCm(), -1000 * 100, 10000 * 100) / 100) + 1000;
+    sbufWriteU16(dst, altitude);
```

Comment:
Can't be an unsigned.

Reply:
Why not? it is converted to meters, constrained -1000m -> +10,000m, then offset 1000m, resulting in a range of 0 -> 11000. 
Since the lowest point on the earth's surface is approx. 413m below sea level, it seems to make sense. 
What did I miss?

Reply:
Ah, ok. I missed the `+ 1000` at the end. Probably a good idea to update the comment then as it says the range is from -1 km to +10 km and doesn't mention anything about the data being sent is offset +1000m.

Reply:
Also note that `getEstimatedAltitudeCm()` is MSL when disarmed. When armed it's relative to the home position. So it's certainly possible for the altitude to be less than -1000m - like if you armed on top of a mountain and flew down.